cmake/PythonNumpyPybind11.cmake

```diff
@@ -51,10 +51,14 @@ execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
         sys.stdout.write(str(sys.version_info[1]))"
         OUTPUT_VARIABLE PYTHON_MINOR_VERSION)
 
+execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
```

Comment:
If we only use this for win32, move it inside the if condition?

Reply:
Resolved